server/plugin/webhook.go

```diff
@@ -459,12 +460,8 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 func (p *Plugin) sanitizeDescription(description string) string {
 	var policy = bluemonday.StrictPolicy()
 	policy.SkipElementsContent("details")
-	// Replacing the unicodes with the respective special characters to have proper rendering in code block.
-	result := strings.ReplaceAll(policy.Sanitize(description), "&#39;", "'")
-	result = strings.ReplaceAll(result, "&#34;", "\"")
-	result = strings.ReplaceAll(result, "&amp;", "&")
-	result = strings.ReplaceAll(result, "&gt;", ">")
-	result = strings.ReplaceAll(result, "&lt;", "<")
+	result := policy.Sanitize(description)
+	html.UnescapeString(result)
```

Comment:
I think we need to reassign the value right?

```suggestion
	result = html.UnescapeString(result)
```